sigridci/sigridci.py

```diff
@@ -243,31 +243,34 @@ def __init__(self, options):
 
     def prepareUpload(self, sourceDir, outputFile):
         zipFile = zipfile.ZipFile(outputFile, "w", zipfile.ZIP_DEFLATED)
+        zipContents = []
         
         for root, dirs, files in os.walk(sourceDir):
             for file in sorted(files):
                 filePath = os.path.join(root, file)
                 if file != outputFile and not self.isExcluded(filePath):
                     relativePath = os.path.relpath(os.path.join(root, file), sourceDir)
                     uploadPath = self.getUploadFilePath(relativePath)
+                    zipContents.append(uploadPath)
```

Comment:
Using a list seems a bit overkill here. Just using a boolean `zipHasContent` which you set to `false` at 246 and `true` here does the trick just as well right? Or do you expect we want to do more with that list of paths later?